torch/utils/backcompat/broadcast/warning/__init__.py

```diff
@@ -0,0 +1,18 @@
+import sys
+from torch._C import _set_backcompat_broadcast_warn
+from torch._C import _get_backcompat_broadcast_warn
+
+
+class Warning:
+    def __init__(self):
+        pass
+
+    def set_enabled(self, value):
+        _set_backcompat_broadcast_warn(value)
+
+    def get_enabled(self):
+        return _get_backcompat_broadcast_warn()
+
+    enabled = property(get_enabled, set_enabled)
+
+sys.modules[__name__] = Warning()
```

Comment:
I meant that you could do this instead of hacking `sys.modules`:
```python
# torch/utils/backcompat/broadcast.py
class Warning(object):
    ... # declare the property here
warning = Warning()
```
Also, maybe let's just have `torch.utils.backcompat.broadcast_warning` and `torch.utils.backcompat.keepdim_warning`? You could reuse the same class for this (just give the getter/setter C function to the constructor of both instances).